docs/src/app/(public)/page.tsx

```diff
@@ -5,18 +5,19 @@ import MailingList from '../../components/mailing-list';
 import CallToAction from '../../components/call-to-action';
 import Footer from '../../components/footer';
 import Features from '../../components/features';
+import { Main } from '../../components/main';
 
 export default function Homepage() {
   return (
     <>
-      <main>
+      <Main isFocusable={false}>
```

Comment:
I'm not sure why but this is the only page that scrolls to the `<main>` on client side nav so setting `isFocusable` to `false` prevents adding `tabindex="-1"` and fixes the issue. The `scroll={false}` prop on `next/link` doesn't work ATM but maybe that's a better fix later on.